templates/docs/patterns/hero/index.md

```diff
@@ -0,0 +1,105 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Hero | Patterns
+---
+
+A hero is a prominent banner section typically used to quickly capture the user's attention after they land on the page.
+Depending on the size and composition of your content, you can choose from a variety of hero layouts:
+
+- [50/50](#5050)
+- [50/50 with full-width image](#5050-with-full-width-image)
+- [25/75 Signpost](#2575-signpost)
+- [75/25](#7525)
+- [Fallback](#fallback) (default)
```

Comment:
Feels bit weird do call "fallback" a default. What does it even mean. If it's a default, why do we call it "fallback" and document last?

To my understanding this is the "last resort" kind of hero: if content (long title) doesn't fit into any of the standard hero layouts.


```suggestion
- [Fallback](#fallback)
```